share/spack/gitlab/cloud_pipelines/configs/ci.yaml

```diff
@@ -110,6 +110,8 @@ ci:
         CI_JOB_SIZE: "small"
         KUBERNETES_CPU_REQUEST: "500m"
         KUBERNETES_MEMORY_REQUEST: "500M"
```

Comment:
Probably not, but also I did a query on the noop jobs and got totals from the beginning of when we started collecting this data, we spend about %0.7 ratio of noop to non-noop jobs.

Some other things we could probably do here to decrease resource usage is not clone the repo or run any of the before script work on these jobs. We could also probably swap out the image for a busybox image startup time to almost 0. These two things would likely gut the total time spent on these jobs by a lot.

Reply:
those improvements sound great 

What I’m mostly worried about is jobs requesting x amount of memory that they won’t use, blocking other real jobs from being allocated to that node. I guess these jobs are quite short but it can’t hurt to optimize anyway.